docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.8"
+services:
+  db:
+    image: postgres:14.3-alpine
```

Comment:
Can you also make postgres restart always? Currently when I restart my computer everything is ready to go except the db.